### ISSUE TITLE
getter 'topRadius' was called on null

### ISSUE THREAD

**Issue by beckythesooh** [jamesblasco/modal_bottom_sheet#89]
Hi, I'm calling CupertinoScaffold.showCupertinoModalBottomSheet in order to create the Cupertino animation. When running the app, the first time I click it, it actually works fine, but the second time I get the error: 

> Unhandled Exception: NoSuchMethodError: The getter 'topRadius' was called on null.
<br>Receiver: null
<br>Tried calling: topRadius

The error goes back to the following line so I assume it has to do with an error in the context.
```
final topRadius = CupertinoScaffold.of(context).topRadius;
```

My code is as below:

```
void _onTap(int index) => {
    setState(() {

      if (_currentIndex != index) {
        // If tapped on new tab, go to new page
        if (index == 1) {
          CupertinoScaffold.showCupertinoModalBottomSheet(
              context: context,
              expand: true,
              duration: Duration(milliseconds: 250),
              builder: (BuildContext context, ScrollController scrollController) => InviteScreen()
          );
        }
        else {
          _pageController.jumpToPage(index);
          _currentIndex = index;
        }
      }

    })
  };
```

Would appreciate any help!

**Comment by attex** [comment]
@beckythesooh 
Hi, what excatly was your problem in the Navigator?

I am facing the same issue.

**Comment by beckythesooh** [comment]
@attex I'm still a beginner so I'm not 100% sure, but basically I was messing up the stack after I closed the popup, so context wasn't being pushed upon the second call to CupertinoScaffold. I was trying to replace the screen underneath the modal and then close the modal: home --> popup overlay --> new page replaces home upon closing the popup --> back button to home. I realized my stack was messed up because I could call the popup multiple times if I never went through that route.
